wasm/internal/spectests/spec_test.go

```diff
@@ -294,10 +294,8 @@ func runTest(t *testing.T, newEngine func() wasm.Engine) {
 							}
 							inst, ok := store.ModuleInstances[moduleName]
 							require.True(t, ok, msg)
```

Comment:
let's add store.GetModuleInstance as well or return *ModuleInstance on store.Instantiate to line with moduleInstance.GetExport API 😄

Reply:
I would like to clean this up further in #170 once done the parser.. I've added it to the list and thanks for noticing!